app/src/main/java/org/oppia/android/app/devoptions/marktopicscompleted/MarkTopicsCompletedFragmentPresenter.kt

```diff
@@ -21,7 +21,8 @@ class MarkTopicsCompletedFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
   private val fragment: Fragment,
   private val viewModelProvider: ViewModelProvider<MarkTopicsCompletedViewModel>,
-  private val modifyLessonProgressController: ModifyLessonProgressController
+  private val modifyLessonProgressController: ModifyLessonProgressController,
+  private val singleTypeAdapterFactory: BindableAdapter.SingleTypeBuilder.Factory
```

Comment:
For consistency: `singleTypeBuilderFactory`

Reply:
Noted.

Reply:
This appears addressed in the latest code. Marking as resolved since Rajat has already approved the PR.